File: palindrome.c
#include<stdio.h>
int main()
{
	int str,r,sum=0,n,i,temp;
	scanf("%s",str);
	while(str!='\0')
	{
		r=n%10;
		sum=(sum*10)+r;
		n=n/10;
	}
	temp==n;
	if(temp==sum)
	{
		printf("palindrome");
	}
	else
	{
		printf("not palindrome");
	}
}

File: palindrome.py
'''
n=input()
i=0
j=len(n)-1
while(i<j):
    if(n[i]!=n[j]):
        print("not palindrome")
        break;
    else:
        print("palindrome")
        break;
'''
s=input()
i=0
j=len(s)-1
while(i<j):
    if s[i]!=s[j]:
        print("false")
        break
    i+=1
    j-=1
else:
        print("true")